org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/resources/bpmn4tosca/bppmn4tosca.json

```diff
@@ -1,116 +1,193 @@
 [
-  {
```

Comment:
I wonder why this file changed.

Would it be possible to add a second test case (and keep the current test case unchanged?)